regifter.sql

```diff
@@ -9,6 +9,8 @@ CREATE DATABASE regifter;
 
 --
 \echo Create a table called gifts
+
+regifter=# CREATE TABLE gifts (id SERIAL PRIMARY KEY, gift TEXT, giver TEXT, value INT, previously_regifted BOOLEAN);
```

Comment:
come back and clean this line up.  Remember copying from your terminal can bring in unwanted text such as we have here.